src/types/rooms.ts

```diff
@@ -1,20 +1,24 @@
-export interface BaseRoom {
+export enum RoomTypes {
+  unclickable = "UNCLICKABLE",
+}
+
+// @debt We should end up with 1 canonical room type
+export interface Room {
   about: string;
   subtitle: string;
   title: string;
   url: string;
-}
-
-export interface Room extends BaseRoom {
-  attendanceBoost?: number;
   height_percent: number;
   image_url: string;
   isEnabled: boolean;
   width_percent: number;
   x_percent: number;
   y_percent: number;
+  attendanceBoost?: number;
+  type?: RoomTypes;
 }
```

Comment:
Copy/paste this block over the diff here, sorts them better to match the v2 styles/etc.

```typescript
export interface Room {
  type?: RoomTypes;
  title: string;
  subtitle: string;
  url: string;
  about: string;
  x_percent: number;
  y_percent: number;
  width_percent: number;
  height_percent: number;
  isEnabled: boolean;
  image_url: string;
  // Legacy?
  attendanceBoost?: number;
}
```